src/v2/Apps/Order/OrderApp.tsx

```diff
@@ -135,18 +135,22 @@ class OrderApp extends React.Component<OrderAppProps, {}> {
     }
 
     const stripePromise = loadStripe(sd.STRIPE_PUBLISHABLE_KEY)
+
+    const isModal = this.props.match?.location.query.isModal ? true : false
+
     return (
       <SystemContextConsumer>
         {({ isEigen, mediator }) => {
           // @ts-expect-error STRICT_NULL_CHECK
           this.mediator = mediator
+
           return (
             <Box>
               {/* FIXME: remove once we refactor out legacy backbone code.
-                  Add place to attach legacy flash message, used in legacy inquiry flow
-              */}
+                    Add place to attach legacy flash message, used in legacy inquiry flow
+                */}
```

Comment:
Is this comment and`<div id="main-layout-flash" />` still needed?

Reply:
We're still using the legacy inquiry flow so I think it has to stay put for now

Reply:
We still use the legacy inquiry flow, so I think we should probably leave it in for now